src/Spatial/Units/Angle.cs

```diff
@@ -311,36 +311,39 @@ public static Angle ReadFrom(XmlReader reader)
             return reader.ReadElementAs<Angle>();
         }
 
-        /// <inheritdoc />
+        /// <summary>
+        /// Returns a string representation of an Angle according to the default representation provided by <see cref="AngleFormatProvider"/>
+        /// </summary>
+        /// <returns>The string representation of this instance.</returns>
         public override string ToString()
         {
-            return this.ToString(null, NumberFormatInfo.CurrentInfo);
+            return this.ToString("R");
         }
 
         /// <summary>
-        /// Returns a string representation of the Angle using the provided format
+        /// Returns a string representation of the Angle using requested format.  See <see cref="AngleFormatProvider"/> for details of formatting options
         /// </summary>
-        /// <param name="format">a string indicating the desired format of the double.</param>
+        /// <param name="format">A string indicating the desired format</param>
```

Comment:
I think the old version is clearer, maybe it can be made even clearer.

Reply:
the old version returned the format of the double - the new one returns the format of the angle - they are not the same thing in this implementation